pkg/cmd/openshift-tests/images/images_command.go

```diff
@@ -83,6 +83,11 @@ func NewImagesCommand() *cobra.Command {
 			for _, line := range lines {
 				fmt.Fprintln(os.Stdout, line)
 			}
+			// TODO: these should be removed when landing k8s 1.31:
```

Comment:
1.32?

Reply:
oh yeah, my bad. Copy-paste mistake...

I'll just live as is to avoid re-scheduling all the jobs again. This will be reverted anyway.